frame/contracts/src/exec.rs

```diff
@@ -315,6 +315,16 @@ pub trait Ext: sealing::Sealed {
 
 	/// Get a mutable reference to the nested gas meter.
 	fn gas_meter(&mut self) -> &mut GasMeter<Self::T>;
+
+	/// Append a string to the debug buffer.
+	///
+	/// It is added as-is without any additional new line.
+	///
+	/// This is a no-op if debug message recording is disabled which is always the case
+	/// when the code is executing on-chain.
+	///
+	/// Returns `true` if debug message recording is disabled. Otherwise `false` is returned.
```

Comment:
If I read implementation correctly
```suggestion
	/// Returns `true` if debug message recording is enabled. Otherwise `false` is returned.
```